migrations/20180701155524-create-user.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
Is this file need?
If no just delete this, pls

Reply:
Done.